dev-python/setuptools_scm/setuptools_scm-7.1.0.recipe

```diff
@@ -25,27 +25,30 @@ BUILD_REQUIRES="
 PYTHON_PACKAGES=(python38 python39 python310)
 PYTHON_VERSIONS=(3.8 3.9 3.10)
 for i in "${!PYTHON_PACKAGES[@]}"; do
-pythonPackage=${PYTHON_PACKAGES[i]}
-pythonVersion=${PYTHON_VERSIONS[$i]}
-eval "PROVIDES_${pythonPackage}=\"\
-	${portName}_$pythonPackage = $portVersion\n\
-	\"; \
-REQUIRES_$pythonPackage=\"\
-	haiku\n\
-	setuptools_$pythonPackage\n\
-	toml_$pythonPackage\n\
-	cmd:git\n\
-	cmd:python$pythonVersion\n\
-	\""
-BUILD_REQUIRES="$BUILD_REQUIRES
-	setuptools_$pythonPackage"
-BUILD_PREREQUIRES="$BUILD_PREREQUIRES
-	cmd:python$pythonVersion"
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_${pythonPackage}=\"
+		${portName}_$pythonPackage = $portVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		haiku
+		setuptools_$pythonPackage
+		toml_$pythonPackage
+		cmd:git
+		cmd:python$pythonVersion
+		\""
+	BUILD_REQUIRES="$BUILD_REQUIRES
+		packaging_$pythonPackage
+		setuptools_$pythonPackage
+		"
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+		cmd:python$pythonVersion
+		"
 done
 
 INSTALL()
 {
-	export SETUPTOOLS_SCM_PRETEND_VERSION="$portVersion"
```

Comment:
This doesn't do or fix anything regarding depending recipes, they should need this variable.